src/applications/dhp-connected-devices/components/DeviceConnectionAlerts.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export const DeviceConnectionSucceededAlert = () => {
+  return (
+    <div data-testId="success-alert">
+      <va-alert
+        close-btn-aria-label="Close notification"
+        status="success"
+        visible
+      >
+        <h3 slot="headline">Device Connected</h3>
```

Comment:
Totally looks fine to me, but the wire frames do show this as `Device connected`, not `Device Connected`.

Reply:
Let's use "Device connected" In general, VA uses sentence case.